pkgs/bundle-image-tarball/default.nix

```diff
@@ -1,11 +1,18 @@
-{runCommand, lib, revstring, bundle-image, pigz }:
+{system, bash, lib, bundle-image, revstring, coreutils, gnutar, pigz} :
 
-let
-  label = "nixmodules-${revstring}";
-in
-
-runCommand label {} ''
-  echo "making tarball..."
-  mkdir -p $out
-  tar --use-compress-program=${pigz}/bin/pigz -Scf $out/disk.raw.tar.gz -C ${bundle-image} disk.raw
-''
+derivation {
```

Comment:
Is derivation a "statement"? Haven't seen it before.

Reply:
It's a builtin function https://nixos.org/manual/nix/stable/language/derivations.html